File: ejemplos/calcular-si-es-horario-laboral/codigo.js
exports.main = async (event, callback) => {
    const now = new Date();
    const utcHour = now.getUTCHours();
    const utcDay = now.getUTCDay();
    const localHour = (utcHour - 5 + 24) % 24;
    const localDay = (utcHour - 5 < 0) ? (utcDay + 6) % 7 : utcDay;
    callback({
        outputFields: {
            es_laboral: localDay >= 1 && localDay <= 5 && localHour >= 9 && localHour < 18
        }
    });
}

File: ejemplos/calcular-si-es-horario-laboral/codigo.py
from datetime import datetime
from zoneinfo import ZoneInfo

def main(event):
  now = datetime.now(ZoneInfo("Etc/GMT+5"))
  current_day = now.weekday()
  current_hour = now.hour
  return {
    "outputFields": {
      "es_laboral": 0 <= current_day <= 4 and 9 <= current_hour < 18
    }
  }